Code/GraphMol/MolDraw2D/DrawTextQt.cpp

```diff
@@ -0,0 +1,111 @@
+//
+//  Copyright (C) 2020 Greg Landrum and T5 Informatics GmbH
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+//
+
+#include <GraphMol/MolDraw2D/MolDraw2D.h>
+#include <GraphMol/MolDraw2D/DrawTextQt.h>
+#include <GraphMol/MolDraw2D/MolDraw2DDetails.h>
+#include <QPainter>
+#include <QCoreApplication>
+
+using namespace std;
+
+namespace RDKit {
+
+// ****************************************************************************
+DrawTextQt::DrawTextQt(double max_fnt_sz, double min_fnt_sz, QPainter &qp)
+    : DrawText(max_fnt_sz, min_fnt_sz), d_qp(qp) {
+  PRECONDITION(
+      QCoreApplication::instance(),
+      "need a global QGuiApplication instance to use the Qt font system");
```

Comment:
I think I've usually seen "QApplication" in this error message :)

Reply:
sorry, I don't follow you here.